products/windows-server-core.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Microsoft Windows Server Core
+category: os
+tags: microsoft windows
+iconSlug: windows
+permalink: /windows-server-core
+alternate_urls:
+  - /windowsservercore
+versionCommand: winver
+releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows%20Server
```

Comment:
Is there an official lifecycle link for Windows Server Core?

Reply:
See reply on Nano Server